### DISCUSSION TITLE
GitHub page not indexed on Google YEARS later

### DISCUSSION THREAD

**Discussion by avi-amalanshu** [community/community#60427] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I first made the repo for my GitHub page in December 2020, and each iteration has had my name in some capacity or the other. My latest commit was in May 2023. Google still does not index my page and it does not show up when you search my name "Avi Amalanshu". I'm certainly not optimizing SEO here, but my website doesn't show up on Google as deep as I can be bothered to check. There are endless (seemingly) irrelevant entries (I doubt anyone else in the world has my name anyway). This has to be some kind of bug, right? Anyone experienced this before?

**Comment by yoannchaudet** [comment]
Your page looks indexed to me:

<img width="1019" alt="image" src="https://github.com/community/community/assets/14911070/4a310a07-c403-4543-bbbc-28c55da28ccf">

It's a Google thing. You can certainly manage your SEO from their console. https://search.google.com/search-console/about

**Reply by avi-amalanshu** [reply]
Looks like it is currently in the process of being indexed, since I couldn't see it under an hour ago. Google Console does the trick-- after 2+ years of not bothering to index, took just over a day. Thanks!
